test/functional/apps/visualize/_linked_saved_searches.js

```diff
@@ -76,7 +79,9 @@ export default function ({ getService, getPageObjects }) {
         await filterBar.toggleFilterEnabled('extension.raw');
         await PageObjects.header.waitUntilLoadingHasFinished();
         const unfilteredData = await PageObjects.visualize.getTableVisData();
-        expect(unfilteredData.trim()).to.be('1,293');
+        retry.waitFor('wait for count to change to 1,293', async () => {
+          expect(unfilteredData.trim()).to.be('1,293');
```

Comment:
just like above, this needs to re-execute `PageObjects.visualize.getTableVisData()` inside the retry, and I'd suggest moving the `.trim()` call into the `getTableVisData()` method so that we don't always have to remember to call `.trim()` when using it.